src/ln/channelmanager.rs

```diff
@@ -4167,6 +4167,27 @@ mod tests {
 		nodes
 	}
 
+	macro_rules! display_txn {
```

Comment:
Hmm, why bother adding this (it looks unused atm)? Can't we just "{:?}" the transaction itself, or did you write this and accidentally commit it?